sentry_sdk/integrations/django/__init__.py

```diff
@@ -7,6 +7,19 @@
 from django import VERSION as DJANGO_VERSION
 from django.core import signals
 
+try:
+    import psycopg2.sql
+
+    def sql_to_string(sql):
+        if isinstance(sql, psycopg2.sql.SQL):
+            return sql.string
+        return sql
+except ImportError:
+    def sql_to_string(sql):
+        return sql
```

Comment:
I'd prefer the full try/except/else blocks so there's less in the try:
```
try:
    import psycopg2.sql
except ImportError:
    def sql_to_string(sql):
        return sql
else:
    # Handle psycopg2 sql object strings (or some such comment)
    def sql_to_string(sql):
        if isinstance(sql, psycopg2.sql.SQL):
            return sql.string
        return sql
```

Reply:
it's fine, just a function declaration whose errors are caught